muikku-timed-notifications-plugin/src/main/java/fi/otavanopisto/muikku/plugins/timed/notifications/NotificationController.java

```diff
@@ -96,9 +136,9 @@ public void sendNotification(String category, String subject, String content, Us
           category,
           subject,
           content,
-          Arrays.asList(recipient)
+          recipients
```

Comment:
cc or bcc for guider might be more appropriate but guess it's ok like this also. Obviously which ever the client prefers.